tests/test_facebook_attribution_window.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from tap_tester import runner, connections
+
+from base import FacebookBaseTest
+
+class FacebookAttributionWindow(FacebookBaseTest):
+
+    # set attribution window
+    attrubution_window = 7
+
+    @staticmethod
+    def name():
+        return "tap_tester_facebook_attribution_window"
+
+    def get_properties(self, original: bool = True):
```

Comment:
Why do we need this method here, isn't it defined in the base.py?

Reply:
Yes, it is defined, but here we wanted to use different values of attribution window.

Reply:
Then can't we add one more optional parameter?

Reply:
No, [here](https://github.com/stitchdata/tap-tester/blob/1bf07330f10b4acbcd18fb1c95b7fd87ef876d16/tap_tester/backends/in_memory.py#L286) only 1 argument can be passed in the function.

Reply:
Okay

Reply:
This is an unfortunate restriction of the tap-tester backend.